### PR TITLE
[WIP] Switch to jekyll-paginate-v2.

### LINKED ISSUE #83: Inlude page number in title et all

### PR DESCRIPTION
**Pull request by XhmikosR**
jekyll-paginate is unmaintained and has quite a few bugs.

TODO:

- [ ] figure out why the paginated pages are not in the sitemap; upstream issue, waiting for a new release
- [x] fix paginated pages title
- [ ] use trails?

Fixes #83

### CODE REVIEW

**Comment by ashmaroli** [comment]
@mdo by "current",  are you referring to the production deploy or the proposed Netlify-deploy as of this writing..?

**Comment by mdo** [comment]
If deploys are as easy with Netlify upon merging PRs, it's probably fine to make the change! Only concerns I'd have are around unfamiliarity with it and my inability to step in and make changes from a GitHub staff perspective (like we have now with Pages).

**Comment by XhmikosR** [comment]
I haven't used Netlify for deployments either myself, with Cloudflare behind it.

It should be fine deployment-wise, just a command. Feel free to add me to the Netlify account and I can have a look.

Otherwise, if GH could add the gem to their gem, we could just keep using GH Pages for prod deployments.

File: _config.yml [hunk 1]
```diff
@@ -6,7 +6,14 @@ future:         true
 permalink:      pretty
 
 # Pagination
-paginate:       5
+pagination:
+  enabled:          true
```
Comment (ashmaroli) [inline _config.yml]: indentation of values have been reduced to start from col 17 on `gh-pages`...

**Comment by XhmikosR** [comment]
I think this should work OK now. They removed v2.1.0 from RubyGems so I went with v2.0.0.

Can you guys (@mdo @Johann-S @MartijnCuppens) try this and confirm everything LGTY?

**Comment by XhmikosR** [comment]
Actually, there's an issue still not fixed https://github.com/sverrirs/jekyll-paginate-v2/issues/142

**Comment by XhmikosR** [comment]
This is now on hold because the gem seems unmaintained and also doesn't work with Jekyll 4.0.0, at least at the time of writing.

**Pull request closed without merging**